modules/edu.gemini.seqexec.server/src/main/scala/edu/gemini/seqexec/server/SeqexecEngine.scala

```diff
@@ -100,6 +83,16 @@ object SeqexecEngine {
         toSeqexecEvent(ev)(qs.toQueue.sequences.map(viewSequence))
     }
 
+  def load(q: engine.EventQueue, seqId: SPObservationID): Task[SeqexecFailure \/ Unit] = {
+    val t = EitherT( Task {
+      val odbSeq = ODBProxy.read(seqId)
+      odbSeq.flatMap(s => SeqTranslate.sequence(DhsClientSim(LocalDate.now))(seqId.stringValue(), s))
```

Comment:
`LocateDate.now` does unsafe IO and should be wrapped in a `Task`, otherwise you can have inconsistencies between dates when constructing vs when evaluating. Isn't there some Scalaz library or similar to deal with dates with a `Task` API?